public/docs/_examples/gettingstarted/ts/src/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
you can remove that if its a problem. i think its a good thing, and prevents larger issues later. But we can discuss that more later. For now, I suggest you add the file back and remove the implict any